auth/auth/auth.py

```diff
@@ -221,15 +221,15 @@ async def get_healthcheck(_) -> web.Response:
 
 @routes.get('')
 @routes.get('/')
-@auth.maybe_authenticated_user
 @web_security_headers
+@auth.maybe_authenticated_user
```

Comment:
this (and other annotation-order changes below) are to make this wrapper happen first (ie be outermost)